### PR TITLE
Add option for FTP buffer size:

### PR DESCRIPTION
**Pull request by zcqian**
- use blocksize option from ftplib, change formatting of line

- implement function for optimal block_size lookup

- added list of known optimal sizes

Resolves Issue #117

### CODE REVIEW

**Review by newgene — COMMENTED** [review]
looks good to me. Before merging, let me doublecheck: are your code changes including anytime incompatible with Python 3.6? At this point, we still want to keep python 3.6 as our minimal supported Python version.

**Comment by zcqian** [comment]
The code was tested on Python 3.6, so it should be compatible.

File: biothings/hub/dataload/dumper.py [hunk 1]
```diff
@@ -483,10 +486,26 @@ class FTPDumper(BaseDumper):
...
 
     # TODO: should we add a __del__ to make sure to close ftp connection ?
     # ftplib has a context __enter__, but we don't use it that way ("with ...")
 
+    def _get_optimal_buffer_size(self) -> int:
+        if self.BLOCK_SIZE is not None:
+            return self.BLOCK_SIZE
+        # else:
+        known_optimal_sizes = {
+            'ftp.ncbi.nlm.nih.gov': 33554432,
```
Comment (newgene) [inline biothings/hub/dataload/dumper.py:499]: @zcqian add the reference link here to explain why we set 32MB for ncbi ftp server?

**Review by newgene — COMMENTED** [review]
ok, sounds good. Just left a minor comment.

**Pull request merged**